qemu/tests/cfg/timedrift_check_clock_offset.cfg

```diff
@@ -18,6 +18,8 @@
             type = timedrift_check_when_crash
             start_vm = no
             ntp_server = "clock.redhat.com"
+            clock_sync_command = "(service chronyd stop || service ntpdate stop)"
```

Comment:
`systemctl` for RHEL.7,8 and `service` for RHEL.6

Reply:
From my checking, `service` is supported on rhel6, rhel7 and rhel8, and

> The service command is a wrapper script that allows system administrators to start, stop, and check the status of services without worrying too much about the actual init system being used. Prior to systemd's introduction, it was a wrapper for /etc/init.d scripts and Upstart's initctl command, and now it is a wrapper for these two and systemctl as well.

so I think it is ok to use `service` on RHEL7, 8.

Reply:
@sitoliu It does work now on RHEL8, but project lead recommended us using systemctl for RHEL8, since it has chance for `service` to be totally replaced by `systemctl`. Let's just follow it. Thanks.